lib/field-maps.js

```diff
@@ -41,26 +41,164 @@ exports.transactions = ({
   masterpassCard,
   riskData,
   getGateway,
-  ...fields
-}) => ({
-  ...fields,
```

Comment:
Braintree API added more fields, which added more fields to the field map, which broke the import because those fields don't exist in the table schema. The fix is to explicitly state all the fields we care about, and ignore all other fields.